etc/nsjail/execute.cfg

```diff
@@ -0,0 +1,168 @@
+name: "compiler explorer compilation/tool sandbox"
+
+mode: ONCE
+hostname: "ce"
+
+time_limit: 0
+
+log_level: INFO
+
+rlimit_as_type: INF
+rlimit_cpu_type: SOFT
+rlimit_fsize: 1024 # 1GB
+rlimit_nofile: 100
+
+uidmap {
+    inside_id: "10240"
+}
+
+gidmap {
+    inside_id: "10240"
+}
+
+# must run following as root during system startup
+# cgcreate -a ubuntu:ubuntu -g memory,pids,cpu,net_cls:ce-compile
+cgroup_mem_parent: "ce-compile"
+cgroup_pids_parent: "ce-compile"
+cgroup_net_cls_parent: "ce-compile"
+cgroup_cpu_parent: "ce-compile"
+
+cgroup_mem_max: 1342177280 # 1.25 GiB
+cgroup_pids_max: 16
+cgroup_cpu_ms_per_sec: 1000
+
+keep_env: true
+# Set the home directory
+envar: "HOME=/app"
+# Remove some env vars, mostly to stop people emailing me about them
+# SUDO_COMMAND is one with actual somewhat sensitive info
+envar: "SUDO_COMMAND="
+envar: "SUDO_USER="
+envar: "SUDO_UID="
+envar: "SUDO_GID="
+envar: "DBUS_SESSION_BUS_ADDRESS="
+
+mount {
+    src: "/bin"
+    dst: "/bin"
+    is_bind: true
+}
+
+mount {
+    src: "/lib"
+    dst: "/lib"
+    is_bind: true
+}
+
+mount {
+    src: "/usr"
+    dst: "/usr"
+    is_bind: true
+}
+
+mount {
+    src: "/etc/localtime"
+    dst: "/etc/localtime"
+    is_bind: true
+}
+mount {
+    src_content: "ce-user:x:10240:10240:Not a real account:/app:/bin/bash"
+    dst: "/etc/passwd"
+    is_bind: true
+}
+
+mount {
+    src: "/lib64"
+    dst: "/lib64"
+    is_bind: true
+    mandatory: false
+}
+
+mount {
+    src: "/lib32"
+    dst: "/lib32"
+    is_bind: true
+    mandatory: false
+}
+
+mount {
+    dst: "/tmp"
+    fstype: "tmpfs"
+    options: "size=134217728,nr_inodes=100" # 128MiB
+    rw: true
+    noexec: true
+    nodev: true
+    nosuid: true
+}
+
+mount {
+    dst: "/dev"
+    fstype: "tmpfs"
+}
+
+mount {
+    src: "/dev/null"
+    dst: "/dev/null"
+    rw: true
+    is_bind: true
+}
+
+mount {
+    src: "/dev/zero"
+    dst: "/dev/zero"
+    is_bind: true
+}
+
+mount {
+    src: "/dev/urandom"
+    dst: "/dev/random"
+    is_bind: true
+}
+
+mount {
+    src: "/dev/urandom"
+    dst: "/dev/urandom"
+    is_bind: true
+}
+
+mount {
+    dst: "/proc"
+    fstype: "proc"
+}
+
+mount {
+    src: "/opt/compiler-explorer"
+    dst: "/opt/compiler-explorer"
+    is_bind: true
+}
+
+# Needed for local libc
+mount {
```

Comment:
I think this is redundant now that /usr is mounted.

Reply:
Good point! fixed.